libr/util/udiff.c

```diff
@@ -239,11 +239,22 @@ R_API bool r_diff_buffers_distance_levenstein(RDiff *d, const ut8 *a, ut32 la, c
 	stop = bLen;
 	// Preliminary tests
 
-	//Do we have both files a & b, and are they at least one byte?
-	if (!aBufPtr || !bBufPtr || aLen < 1 || bLen < 1) {
+	//Do we have both files a & b
+	if (!aBufPtr || !bBufPtr) {
```

Comment:
uhm, it is valid to pass those nulls? shouldnt be asserting because of bad api usage?

Reply:
I agree, I updated to check that at function entry with `r_return_val_if_fail`.